source/assets/public/index.css

```diff
@@ -827,3 +705,158 @@ p.c-price-list__description {
 .h-centred {
   text-align: center;
 }
+
+/* phone */
+@media (max-width: 639px) {
+  .nav--header {
+    display: none;
+  }
+  .nav__room {
+    padding-top: 4em;
+  }
+  
+  .nav--mobile .nav__content {
+    background: var(--primary-contrast);
+    width: 100%;
+    padding-top: 6rem;
+    padding-bottom: 3rem;
+    z-index: 5;
+    text-align: left;
+  }
+
+  .nav.nav--mobile {
+    position: fixed;
+    padding: 0;
+    margin: 0;
+    top: 0;
+    right: 0;
+    left: 0;
+  }
+  /* the new stacking context created by the transform */
+  /* was giving me grief with the mobile nav */
+  .hero__date {
+    transform: none;
+  }
+  .hero {
+    width: 94%;
+  }
+    .hero__logo {
+      max-width: 70%;
+      margin-left: auto;
+      margin-right: auto;
+    }
+    .hero__slogan {
+      font-size: 3.3rem;
+      line-height: 4rem;
+      text-align: left;
+    }
+      .hero__slogan span {
+        font-size: 3rem;
+        line-height: 4rem;
+      }
+
+  .c-sponsor__logo {
```

Comment:
It's very difficult to review this diff since it's so big, might be worth moving some things into partials here

Reply:
How about I split this PR up into smaller PRs?

Reply:
Yep that sounds good - I'm happy to merge the fontawesome part, not 100% confident on the other two parts, would like to review the css more closely for the switch from metaquery to @media query